packages/fluentui/ability-attributes/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "repository": "microsoft/fluent-ui-react.git",
   "scripts": {
-    "build": "npm run schema && gulp bundle:package:no-umd",
+    "build": "yarn schema && gulp bundle:package:no-umd",
```

Comment:
TODO: If this change needs to be kept it needs to be made part of `scripts/fluent-import/index.js`.